docker-images/kafka/kafka-thirdparty-libs/2.3.x/pom.xml

```diff
@@ -107,5 +107,35 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>io.strimzi</groupId>
+            <artifactId>kafka-oauth-keycloak-authorizer</artifactId>
+            <version>${strimzi-oauth-callback.version}</version>
```

Comment:
Should we change the name of the property? It is not just cllback anymore, so maybe `strimzi-oauth.version` would be better